### PR TITLE
Workflow's name is required, so add it to examples

### PR DESCRIPTION
**Pull request by fiskus**
# Description

# TODO

- [ ] ~Unit tests~
- [ ] ~Automated tests (e.g. Preflight)~
- [x ] Documentation
    - [ ] ~[Python: Ru `build.py`](../gendocs/build.py) for new docstrings~
    - [ ] ~JavaScript: basic explanation and screenshot of new features~
- [ ] ~[Changelog](CHANGELOG.md) entry~

### CODE REVIEW

File: docs/advanced-features/workflows.md [hunk 1]
```diff
@@ -299,8 +301,10 @@ objects known as *package entries*. Each package entry defines a logical key
 ```yaml
 workflows:
   myworkflow-1:
+    name: My workflow №1
     entries_schema: must-contain-readme
   myworkflow-2:
+    name: My workflow №2
```
Comment (nl0) [inline docs/advanced-features/workflows.md:307]: in English using `#` is more common than `№`

**Review by nl0 — APPROVED** [review]

File: docs/advanced-features/workflows.md (hunk 1 above)
Comment (fiskus) [reply docs/advanced-features/workflows.md:307]: Yes, but wouldn't it be treated as comment in YAML?

Comment (sir-sigurd) [reply docs/advanced-features/workflows.md:307]: I think you can quote the string:
```suggestion
    name: 'My workflow #2'
```

File: docs/advanced-features/workflows.md [hunk 2]
```diff
@@ -299,8 +301,10 @@ objects known as *package entries*. Each package entry defines a logical key
 ```yaml
 workflows:
   myworkflow-1:
+    name: My workflow №1
```
Comment (sir-sigurd) [inline docs/advanced-features/workflows.md:304]: What about this one?

Comment (fiskus) [reply docs/advanced-features/workflows.md]: Fixed

**Review by sir-sigurd — APPROVED** [review]

**Pull request merged**